extras/signers.json

```diff
@@ -44,6 +44,21 @@
     "nanexcool": "0x823DF0278e4998cD0D06FB857fBD51e85b18A250",
     "davgarai": "0xAc1aA53108712d7f38093A67d380aD54B562a650"
   },
+  "arb_grant_treasury": {
+    "balancer": {
+      "tritium": "0xcf4fF1e03830D692F52EB094c52A5A6A2181Ab3F",
+      "mikeb": "0xc4591c41e01a7a654B5427f39Bbd1dEe5bD45D1D",
+      "xeonus": "0xb66941ffab0bad8f118c022921d8ac6f238c2eeb"
+    },
+    "aura": {
+      "jack": "0x374631880f03Ee04bFa263A30b72affCCE0c8DAf",
+      "json": "0xE2A4DE267cdD4fF5ED9Ba13552F5c624b12db9b2",
+      "lamentations": "0x196421298af44afb88b978fbd333dc49218b3d38"
+    },
+    "emergency": {
+      "arb_foundation_treasury": "0xf3fc178157fb3c87548baa86f9d24ba38e649b58"
```

Comment:
See etherscan: https://arbiscan.io/address/0xf3fc178157fb3c87548baa86f9d24ba38e649b58

Note we did not ask the arb treasury multisig for this, and it would only come into play if a number of signers lost keys and we were trying to rescue funds, but 4/6 is a little tight.